src/shared/strengths-weaknesses/StrengthsOrWeaknesses.tsx

```diff
@@ -12,37 +12,60 @@ import LimitedTextAreaInput from '../forminator/inputs/LimitedTextAreaInput';
 interface OwnProps {
   title: string;
   maxLength: number;
+  label?: string;
 }
 
 type Props = FCProps<OwnProps>;
 
-export function StrengthsOrWeaknesses({ title, maxLength, ...props }: Props) {
+export function StrengthsOrWeaknesses({ title, maxLength, label, ...props }: Props) {
   const lens = useFragmentLens();
-  const condition = useCallback(
+  const firstCondition = useCallback((value: unknown[]) => {
+    return value ? value.length < 2 : true;
+  }, []);
+  const secondCondition = useCallback(
     (value: unknown[]) => {
-      return value ? value.length < maxLength : true;
+      return value ? value.length > 1 && value.length < maxLength : false;
     },
     [maxLength],
   );
   return (
     <Card>
-      <CardHeader title={title} />
+      <CardHeader title={title} titleTypographyProps={{ variant: 'h6' }} />
       <CardContent>
-        <Grid container spacing={2}>
-          <ArrayInput initialValue={[undefined]}>
-            <FragmentRef lens={lens} />
-            <ArrayOutput>
-              <Grid item xs={12}>
-                <LimitedTextAreaInput variant="outlined" maxChars={280} fullWidth inputProps={{ dir: 'auto' }} />
-              </Grid>
-            </ArrayOutput>
-            <ConditionalSection condition={condition} lens={lens}>
-              <Grid item>
-                <ArrayAppendButton>{i18n._('Add')}</ArrayAppendButton>
-              </Grid>
-            </ConditionalSection>
-          </ArrayInput>
-        </Grid>
+        <Container maxWidth="sm">
+          <Grid container spacing={2}>
+            <ArrayInput initialValue={[undefined]}>
+              <FragmentRef lens={lens} />
+              <ArrayOutput>
+                <Grid item xs={12}>
+                  <LimitedTextAreaInput
+                    variant="outlined"
+                    maxChars={280}
+                    fullWidth
+                    inputProps={{ dir: 'auto' }}
+                    label={label}
+                  />
+                </Grid>
+              </ArrayOutput>
+              <ConditionalSection condition={firstCondition} lens={lens}>
```

Comment:
Is it possible to refactor codes  something like this:
```jsx
<Grid item xs />
<Grid item>
  <ArrayAppendButton variant="outlined" color="primary">
    {firstCondition() && i18n._('Add')  || secondCondition() && i18n._('What else')}
  </ArrayAppendButton>
</Grid>
```

Reply:
i dont think so cuz the button is not always displayed. we only want to display it when value.length is less than maxLength